components/sections/logoCloud/variant_c.tsx

```diff
@@ -1,136 +1,144 @@
-              />
-            </div>
-          )}
-          {images?.[1]?.image && (
-            <div className="absolute left-0 top-0 mt-20 flex h-24 w-24 items-center justify-center rounded-full bg-gray-50">
-              <Image
-                className="h-16 w-16 object-scale-down"
-                src={urlFor(images[1]?.image)}
-                sizes="100vw"
-                width={64}
-                height={64}
-                alt={images[1]?.alt ?? "logoCloud-image-2"}
-              />
-            </div>
-          )}
-          {images?.[2]?.image && (
-            <div
-              className="absolute flex h-40 w-40 items-center justify-center rounded-full bg-gray-50"
-              style={{ bottom: "-250px", left: "20%" }}
-            >
-              <Image
-                className="h-32 w-32 object-scale-down"
-                src={urlFor(images[2]?.image)}
-                sizes="100vw"
-                width={96}
-                height={96}
-                alt={images[2]?.alt ?? "logoCloud-image-3"}
-              />
-            </div>
-          )}
-          {images?.[3]?.image && (
-            <div
-              className="absolute flex h-40 w-40 items-center justify-center rounded-full bg-gray-50"
-              style={{ top: "20px", right: "20%" }}
-            >
-              <Image
-                className="h-32 w-32 object-scale-down"
-                src={urlFor(images[3]?.image)}
-                sizes="100vw"
-                width={96}
-                height={96}
-                style={{ objectFit: "scale-down" }}
-                alt={images[3]?.alt ?? "logoCloud-image-4"}
-              />
-            </div>
+            </Button>
           )}
-          {images?.[4]?.image && (
-            <div
-              className="absolute flex h-32 w-32 items-center justify-center rounded-full bg-gray-50"
-              style={{ bottom: "-250px", right: 0 }}
-            >
-              <Image
-                className="h-24 w-24 object-scale-down"
-                src={urlFor(images[4]?.image)}
-                sizes="100vw"
-                width={64}
-                height={64}
-                alt={images[4]?.alt ?? "logoCloud-image-5"}
-              />
-            </div>
-          )}
-          {images?.[5]?.image && (
-            <div
-              className="absolute right-0 flex h-24 w-24 items-center justify-center rounded-full bg-gray-50"
-              style={{ top: "-150px" }}
-            >
-              <Image
-                className="h-16 w-16 object-scale-down"
-                src={urlFor(images[5]?.image)}
-                sizes="100vw"
-                width={64}
-                height={64}
-                alt={images[5]?.alt ?? "logoCloud-image-6"}
-              />
-            </div>
-          )}
-        </div>
-        <div className="mt-16 flex flex-wrap justify-center lg:hidden">
+        </Container>
+        <Logos images={images} />
```

Comment:
Kindly conditionally render Logo, Title elements to only display if it is defined/has value. This is to make sure we don't display an `undefined` or throw and error when its values are expected.